### PR TITLE
[WIP] Feature/centos7 sysvm

### PR DESCRIPTION
**Pull request by borisroman**

### CODE REVIEW

File: cosmic-core/systemvm/patches/centos7/opt/cosmic/router/scripts/checks2svpn.sh [hunk 1]
```diff
@@ -6,9 +6,9 @@ then
     exit 1
 fi
 
-if [ -f /etc/cloudstack-release ]
+if [ -f /etc/cosmic-release ]
   then
-    VERSION_DATA=$(cat /etc/cloudstack-release | cut -d" " -f3)
+    VERSION_DATA=$(cat /etc/cosmci-release | cut -d" " -f3)
```
Comment (resmo) [inline cosmic-core/systemvm/patches/centos7/opt/cosmic/router/scripts/checks2svpn.sh]: Typo

File: cosmic-core/systemvm/patches/centos7/opt/cosmic/router/bin/merge.py [hunk 2]
```diff
@@ -191,7 +191,7 @@ def get_macaddress_from_databag(d_to_merge):
 
     def update_dbag_contents(self):
         d_to_merge = self.validate_device_based_on_mac_address()
-        d_ip_to_save = {}
+        d_ip_to_save = { }
```
Comment (resmo) [inline cosmic-core/systemvm/patches/centos7/opt/cosmic/router/bin/merge.py]: This makes it not pep8 compliant.

File: cosmic-core/systemvm/patches/centos7/opt/cosmic/router/bin/merge.py [hunk 3]
```diff
@@ -70,12 +69,12 @@ def setKey(self, key):
...
 
 class updateDataBag:
-    DPATH = "/etc/cloudstack"
+    DPATH = "/etc/cosmic/router"
 
     def __init__(self, qFile):
         self.qFile = qFile
         self.fpath = ''
-        self.bdata = {}
+        self.bdata = { }
```
Comment (resmo) [inline cosmic-core/systemvm/patches/centos7/opt/cosmic/router/bin/merge.py]: This makes it not pep8 compliant.

File: cosmic-core/systemvm/patches/centos7/opt/cosmic/router/scripts/checks2svpn.sh (hunk 1 above)
Comment (borisroman) [reply cosmic-core/systemvm/patches/centos7/opt/cosmic/router/scripts/checks2svpn.sh]: Thanks!

**Pull request closed without merging**